test/utils/shippable/units.sh

```diff
@@ -6,14 +6,55 @@ declare -a args
 IFS='/:' read -ra args <<< "$1"
 
 version="${args[1]}"
+group="${args[2]}"
 
 if [[ "${COVERAGE:-}" == "--coverage" ]]; then
-    timeout=99
+    timeout=60
 else
-    timeout=11
+    timeout=6
```

Comment:
Don't we also need to backport the change that increases the timeout to accommodate the download time?

Reply:
Actually, it looks like that was already backported, so this would be undoing that. I'll remove the timeout changes so they match `devel`.

Reply:
Yes. I'll add it to this PR.